src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -4395,15 +4396,15 @@ internal static IEnumerable<CompletionResult> CompleteFilename(CompletionContext
                 // specified a drive or portion of the path.
                 var executionContext = context.ExecutionContext;
                 var defaultRelative = string.IsNullOrWhiteSpace(wordToComplete)
-                                      || (wordToComplete.IndexOfAny(Utils.Separators.Directory) != 0 &&
+                                      || (wordToComplete.AsSpan().IndexOfAny('\\', '/') != 0 &&
                                           !Regex.Match(wordToComplete, @"^~[\\/]+.*").Success &&
                                           !executionContext.LocationGlobber.IsAbsolutePath(wordToComplete, out _));
                 var relativePaths = context.GetOption("RelativePaths", @default: defaultRelative);
                 var useLiteralPath = context.GetOption("LiteralPaths", @default: false);
 
-                if (useLiteralPath && LocationGlobber.StringContainsGlobCharacters(wordToComplete))
+                if (useLiteralPath)
                 {
-                    wordToComplete = WildcardPattern.Escape(wordToComplete, Utils.Separators.StarOrQuestion);
+                    wordToComplete = WildcardPattern.Escape(wordToComplete, "[]");
```

Comment:
> You point to removed line.
@iSazonov this line

Reply:
@iSazonov Can you please address this comment?

Reply:
Because Escape() method was changed. Previously the argument contains chars to exclude, now - chars to escape.